### ISSUE TITLE
It is impossible to style parent edge separately

### ISSUE THREAD

**Issue by Witko** [ferstl/depgraph-maven-plugin#73]
Im trying to style the edges of the graph but it seems the parent edge is also "included" as the other dependencies. Is it possible to do this?

**Comment by ferstl** [comment]
In the next release you'll be able to style edges from parents to modules:
```
"edge-resolution-styles": {
  ...
  "parent": {
    "style": "dashed"
  },
}
```

**Comment by Witko** [comment]
Hi @ferstl ,
perfect thank you!

**Comment by ferstl** [comment]
Released version 3.2.0 of the plugin
